app/Models/Concerns/HasFilters.php

```diff
@@ -389,9 +389,9 @@ protected function filterTemplate(Builder $query, string $value = null): void
         $query = $this->joinEntity($query);
 
         if ($value) {
-            $query->whereNotNull('e.is_template');
+            $query->where('e.is_template', 1);
         } else {
-            $query->whereNull('e.is_template');
+            $query->whereNull('e.is_template')->orWhere('e.is_template', '<>', 1);
```

Comment:
When I do a filter for name `Adam` + is not template, it's not showing me the results as expected.
![image](https://user-images.githubusercontent.com/3263440/198685287-dc7178d6-96f7-4338-9341-468f623be36a.png)

```sql
select distinct ....
 from `characters`
 left join `entities` as `e` on `e`.`entity_id` = `characters`.`id` and `e`.`type_id` = 1 and e.campaign_id = characters.campaign_id 
where (`characters`.`name` like '%Adam%' and `e`.`is_template` is null or `e`.`is_template` <> 1) 
...
```
It's not separating the two is_template queries into a single () statement